__tests__/workers/userUpdatedCio.ts

```diff
@@ -59,6 +60,12 @@ describe('userUpdatedCio', () => {
       updated_at: 1714577744,
       username: 'cio',
       referral_link: referral,
+      accepted_marketing: true,
+      cio_subscription_preferences: {
+        topics: {
+          topic_4: true,
+        },
+      },
```

Comment:
Maybe we should add a test for false as well, just to verify down the line if breaking stuff comes?

Reply:
Done